bin/version.json

```diff
@@ -1,12 +1,12 @@
 {
   "esVersion": {
-    "previous": "7.7.0",
-    "current":"7.8.0",
-    "next": "7.8.1"
+    "previous": "7.8.0",
+    "current":"7.9.0",
+    "next": "7.9.1"
   }, 
   "odVersion": {
-    "previous": "1.8.0",
-    "current": "1.9.0",
-    "next": "1.9.1"
+    "previous": "1.9.0",
+    "current": "1.10.0",
+    "next": "1.10.1"
```

Comment:
These changes should be in a standalone PR. They are not part of the slack change.

Reply:
Until Next Time 👍